### PR TITLE
ARM: dts: nexus4: initial dts

### PR DESCRIPTION
**Pull request by devbis**
Add initial support for LGE Nexus 4 (mako)
Works:
- regulators
- emmc
- wifi
- lcd backlight with external driver

### CODE REVIEW

File: arch/arm/boot/dts/qcom-apq8064-lge-nexus4-mako.dts [hunk 1]
```diff
@@ -0,0 +1,395 @@
+// SPDX-License-Identifier: GPL-2.0
```
Comment (okias) [inline arch/arm/boot/dts/qcom-apq8064-lge-nexus4-mako.dts]: https://spdx.org/licenses/GPL-2.0-only.html (GPL-2.0 identifier is deprecated)

File: arch/arm/boot/dts/qcom-apq8064-lge-nexus4-mako.dts [hunk 2]
```diff
@@ -0,0 +1,395 @@
...
+		ramoops@88d00000{
+			compatible = "ramoops";
+			reg = <0x88d00000 0x100000>;
+			record-size	 = <0x00020000>;
+			console-size	= <0x00020000>;
+			ftrace-size	 = <0x00020000>;
+		};
+	};
+
+	gpio-keys {
```
Comment (okias) [inline arch/arm/boot/dts/qcom-apq8064-lge-nexus4-mako.dts]: does these keys work?

File: arch/arm/boot/dts/qcom-apq8064-lge-nexus4-mako.dts [hunk 3]
```diff
@@ -0,0 +1,395 @@
...
+				clocks = <&gcc GSBI4_UART_CLK>, <&gcc GSBI4_H_CLK>;
+				clock-names = "core", "iface";
+				status = "okay";
+
+				pinctrl-names = "default";
+				pinctrl-0 = <&gsbi4_uart_pin_a>;
+			};
+		};
+
+		/* OTG */
```
Comment (okias) [inline arch/arm/boot/dts/qcom-apq8064-lge-nexus4-mako.dts]: usb isn't working yet, drop it from DT

**Review by okias — CHANGES_REQUESTED** [review]
overall good, few nitpicks :)

File: arch/arm/boot/dts/qcom-apq8064-lge-nexus4-mako.dts (hunk 1 above)
Comment (devbis) [reply arch/arm/boot/dts/qcom-apq8064-lge-nexus4-mako.dts]: Thanks, fixed it

File: arch/arm/boot/dts/qcom-apq8064-lge-nexus4-mako.dts (hunk 2 above)
Comment (devbis) [reply arch/arm/boot/dts/qcom-apq8064-lge-nexus4-mako.dts]: I'm not sure, didn't test them yet. Removed for consistency.

File: arch/arm/boot/dts/qcom-apq8064-lge-nexus4-mako.dts (hunk 3 above)
Comment (devbis) [reply arch/arm/boot/dts/qcom-apq8064-lge-nexus4-mako.dts]: Yeah, you're right. Removed.

**Review by okias — APPROVED** [review]

**Pull request merged**